exporter/opentelemetry-exporter-zipkin/tests/test_zipkin_exporter.py

```diff
@@ -425,8 +425,23 @@ def test_export_json_max_tag_length(self):
         span.start()
         span.resource = Resource({})
         # added here to preserve order
-        span.set_attribute("k1", "v" * 500)
-        span.set_attribute("k2", "v" * 50)
+        span.set_attribute("string1", "v" * 500)
+        span.set_attribute("string2", "v" * 50)
+        span.set_attribute("list1", ["a"] * 25)
+        span.set_attribute("list2", ["a"] * 10)
+        span.set_attribute("list3", [2] * 25)
+        span.set_attribute("list4", [2] * 10)
+        span.set_attribute("list5", [True] * 25)
+        span.set_attribute("list6", [True] * 10)
+        span.set_attribute("tuple1", ("a",) * 25)
+        span.set_attribute("tuple2", ("a",) * 10)
+        span.set_attribute("tuple3", (2,) * 25)
+        span.set_attribute("tuple4", (2,) * 10)
+        span.set_attribute("tuple5", (True,) * 25)
+        span.set_attribute("tuple6", (True,) * 10)
+        span.set_attribute("range1", range(0, 25))
+        span.set_attribute("range2", range(0, 10))
+
```

Comment:
Could you add a cases for:
- empty sequence
- sequence with `None`s e.g. `["hello", None, "world"]`

Reply:
Added. Note that None will be converted to null in the json output.